native-utils/src/main/java/com/github/weisj/darklaf/platform/NativeUtil.java

```diff
@@ -1,7 +1,7 @@
 /*
  * MIT License
  *
- * Copyright (c) 2020 Jannis Weis
+ * Copyright (c) 2012 Adam Heinrich <adam@adamh.cz>
```

Comment:
@weisJ this is a non-trivial change, and I guess the file is taken as is from Adam Heinrich work. Is it the case?

Reply:
Yes the copyright was automatically generated by intellJ and should definitly read 'AdamHeinrich'